test/e2e/cataloge2e/catalog_update_parallel_test.go

```diff
@@ -24,7 +24,7 @@ var _ = framework.CLICoreDescribe("[Tests:E2E][Feature:Catalog-Update]", func()
 
 			pluginsList, err := framework.GetPluginsList(tf, true)
 			Expect(err).To(BeNil(), "should not get any error for plugin list")
-			Expect(len(pluginsList)).Should(BeNumerically("==", 0), "plugins list should not return any plugins after plugin clean")
+			Expect(len(pluginsList)).Should(BeNumerically("==", 1), "plugins list should return only one essential plugin after plugin clean")
```

Comment:
After this refresh change when we run
tanzu plugin clean
tanzu plugin list (This step will install essential plugins i.e. telemetry for now since the refresh of db runs if there is no digest file found)

Reply:
Oh right, that makes sense.  Thanks